src/js/components/Accordion/doc.js

```diff
@@ -0,0 +1,72 @@
+import { describe, PropTypes } from 'react-desc';
+
+import {
+  getAvailableAtBadge,
+} from '../../utils';
+
+import AccordionPanel from './AccordionPanel';
+
+export function accordionPanel(Panel) {
+  const DocumentedAccordionPanel = describe(Panel)
+    .description(
+      'An Accordion panel.'
+    );
+  DocumentedAccordionPanel.propTypes = {
+    title: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.node,
+    ]).description(
+      'The panel title.'
+    ).isRequired,
+    header: PropTypes.node.description(
+      'If specified, the entire panel header will be managed by the caller.'
+    ),
+  };
+  return DocumentedAccordionPanel;
+}
+
+export default (Accordion) => {
+  const DocumentedAccordion = describe(Accordion)
+    .availableAt(getAvailableAtBadge('Accordion'))
+    .description('An accordion containing collapsible panels.')
+    .usage(
+      `import { Accordion, AccordionPanel } from 'grommet';
+<Accordion>
+  <AccordionPanel title='Panel 1'>...</AccordionPanel>
+  <AccordionPanel title='Panek 2'>...</AccordionPanel>
+</Accordion>`
+    );
+
+  DocumentedAccordion.propTypes = {
+    activeIndex: PropTypes.oneOfType([
+      PropTypes.number,
+      PropTypes.arrayOf(PropTypes.number),
+    ]).description(
+      `Active panel index. If specified, Accordion will be a controlled component. This means that future
+panel changes will not work unless you subscribe to onActive function and update activeIndex
+accordingly.`
+    ).defaultValue(0),
+    animate: PropTypes.bool.description(
+      'Transition content in & out with a slide down animation.'
+    ).defaultValue(true),
+    children: PropTypes.arrayOf(PropTypes.instanceOf(AccordionPanel)).description(
+      'Array of AccordionPanels.'
+    ).isRequired,
+    onActive: PropTypes.func.description(
+      `Function that will be called with the active panel index when the currently active
```

Comment:
Would it make sense to call this with the same thing that can be provided to `activeIndex`? In other words, if `multiple` and a panel is opened or closed, we always deliver an array of the currently active panels?

Reply:
sure I like the suggestion.